src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -546,13 +546,73 @@ public void resetDASHStats(String streamId) {
 		}
 	}
 
+	public void sendStartPlayWebHook(final String streamId, final String viewerId, final String token){
+		final Broadcast broadcast = getDataStore().get(streamId);
+		if(broadcast == null){
+			return;
+		}
+		final String listenerHookURL = broadcast.getListenerHookURL();
+		if (listenerHookURL == null || listenerHookURL.isEmpty()) {
+			return;
+		}
+		final String name = broadcast.getName();
+		final String category = broadcast.getCategory();
+		final String metaData = broadcast.getMetaData();
+		logger.info("Setting timer to call viewer play started hook for stream:{}", streamId);
+		vertx.setTimer(10, e -> notifyHook(listenerHookURL, streamId, HOOK_ACTION_START_PLAY, name, category,
+				null, null, viewerId, token, metaData));
```

Comment:
For this webhook, there should be a protocol parameter. I mean webhook should include the parameter if it's webrtc, rtmp, hls or any other playback.

Reply:
done